pyweather.py

```diff
@@ -1909,7 +1956,156 @@ def radar_clearImages():
+    yesterday_maxDewpointC = str(data['maxdewptm'])
+    yesterday_minDewpointF = str(data['mindewpti'])
+    yesterday_minDewpointC = str(data['mindewptm'])
+    logger.debug("yesterday_maxDewpointF: %s ; yesterday_maxDewpointC: %s" %
+                 (yesterday_maxDewpointF, yesterday_maxDewpointC))
+    logger.debug("yesterday_minDewpointF: %s ; yesterday_minDewpointC: %s" %
+                 (yesterday_minDewpointF, yesterday_minDewpointC))
+    yesterday_maxPressureInHg = str(data['maxpressurei'])
+    yesterday_maxPressureMB = str(data['maxpressurem'])
+    yesterday_minPressureInHg = str(data['minpressurei'])
+    yesterday_minPressureMB = str(data['minpressurem'])
+    logger.debug("yesterday_maxPressureInHg: %s ; yesterday_maxPressureMB: %s" %
+                 (yesterday_maxPressureInHg, yesterday_maxPressureMB))
+    logger.debug("yesterday_minPressureInHg: %s ; yesterday_minPressureMB: %s" %
+                 (yesterday_minPressureInHg, yesterday_minPressureMB))
+    yesterday_maxWindMPH = str(data['maxwspdi'])
+    yesterday_maxWindKPH = str(data['maxwspdm'])
+    yesterday_minWindMPH = str(data['minwspdi'])
+    yesterday_minWindKPH = str(data['minwspdm'])
+    logger.debug("yesterday_maxWindMPH: %s ; yesterday_maxWindKPH: %s" %
+                 (yesterday_maxWindMPH, yesterday_maxWindMPH))
+    logger.debug("yesterday_minWindMPH: %s ; yesterday_minWindKPH: %s" %
+                 (yesterday_minWindMPH, yesterday_minWindKPH))
+    yesterday_maxVisibilityMI = str(data['maxvisi'])
+    yesterday_maxVisibilityKM = str(data['maxvism'])
+    yesterday_minVisibilityMI = str(data['minvisi'])
+    yesterday_minVisibilityKM = str(data['minvism'])
+    logger.debug("yesterday_maxVisibilityMI: %s ; yesterday_maxVisibilityKM: %s" %
+                 (yesterday_maxVisibilityMI, yesterday_maxVisibilityKM))
+    logger.debug("yesterday_minVisibilityMI: %s ; yesterday_minVisibilityKM: %s" %
+                 (yesterday_minVisibilityMI, yesterday_minVisibilityKM))
+    yesterday_precipMM = str(data['precipm'])
+    yesterday_precipIN = str(data['precipi'])
+    logger.debug("yesterday_precipMM: %s ; yesterday_precipIN: %s" %
+                 (yesterday_precipMM, yesterday_precipIN))
+
+    if yesterday_precipMM == "T":
+        yesterday_precipdata = False
+    else:
+        yesterday_precipdata = True
+    logger.debug("yesterday_precipdata: %s" % yesterday_precipdata)
+
+print(Fore.YELLOW + "Minimum Temperature: " + Fore.CYAN + yesterday_minTempF
+      + "°F (" + yesterday_minTempC + "°C)")
+print(Fore.YELLOW + "Average Temperature: " + Fore.CYAN + yesterday_avgTempF
+      + "°F (" + yesterday_avgTempC + "°C)")
+print(Fore.YELLOW + "Maxmimum Temperature: " + Fore.CYAN + yesterday_maxTempF
+      + "°F (" + yesterday_maxTempC + "°C)")
+print(Fore.YELLOW + "Minimum Dew Point: " + Fore.CYAN + yesterday_minDewpointF
+      + "°F (" + yesterday_minDewpointC + "°C)")
+print(Fore.YELLOW + "Average Dew Point: " + Fore.CYAN + yesterday_avgDewpointF

```

Comment:
You'll want to make sure that you add an additional `Style.BRIGHT` after each color change, and on each line.